aten/src/ATen/functorch/BatchRulesModules.cpp

```diff
@@ -74,7 +74,7 @@ embedding_dense_backward_batch_rule(
   // Fill in the padding. We can't do it in the embedding_dense_backward call
   // because we need to fill in multiple rows!
   if (padding_idx >= 0) {
-    result.select(1, padding_idx).fill_(0);
+    result.select_symint(1, padding_idx).fill_(0);
```

Comment:
Had to add `SymInt` support to `select` because of this. See the previous PR in the stack.